### ISSUE TITLE
Default value for use.normal parameter in thresh

### ISSUE THREAD

**Issue by ernesto561** [raff-k/RainSlide#5]
In `RainSlide::thresh` there is an option to avoid fitting to a Gaussian function (`use.normal = FALSE`). If this option is used, the standard error is calculated as `m.PDF.sigma <- sqrt(m.dev/m.res.df) # sqrt(stats::deviance(.)/stats::df.residual(.))`, as shown in [thresh.R#L176](https://github.com/raff-k/RainSlide/blob/a52fd0e31db6104fb7c4c6524d0bb4e3465edd52/R/thresh.R#L176)

 I understand that if I want to get results more similar to the ones by [Rossi et al (2017)](https://doi.org/10.1016/j.geomorph.2017.02.001), [Peruccacci et al (2012)](https://doi.org/10.1016/j.geomorph.2011.10.005) or [Melillo et al (2018)](https://doi.org/10.1016/j.envsoft.2018.03.024), I should set use.normal = TRUE (correct me if I am wrong). Is there any particular reason why this option was added and the default was set as FALSE?  Shouldn't the default be set as TRUE?

**Comment by raff-k** [comment]
You are right about the results using `use.normal = TRUE`. The reason is that I see this approach as a kind of non-efficient detour. Residuals are (approximately) normal distributed. Why then to approximate the residuals with a normal distribution, when there is a direct way to get `sigma`? For me, it introduces more sources of potential uncertainty. - With the alternative, of course, each user can decide for themselves how this is calculated.

_Nevertheless, more package documentation would help^^_

**Comment by ernesto561** [comment]
Thanks for your answer. I understand now. By the way, in the CTRL-T code, the probability density function is estimated with Kernel Density Estimation, but according what I understand, it's not used later (I have seen that you included and commented this line in thresh.R)

As you said, it would be useful to include in the documentation why this option is available.
